src/Symfony/Component/Config/Tests/Definition/Builder/NodeDefinitionTest.php

```diff
@@ -20,20 +20,44 @@ class NodeDefinitionTest extends TestCase
 {
     public function testDefaultPathSeparatorIsDot()
     {
-        $node = $this->getMockForAbstractClass(NodeDefinition::class, array('foo'));
-
-        $this->assertAttributeSame('.', 'pathSeparator', $node);
+        $node = new class('foo') extends NodeDefinition
```

Comment:
hmmm.. now that I think of it... since 3.4 still works on 5 anonymous class must be dropped and the PR must target 3.4 right?